web/src/pages/index.md

```diff
@@ -7,36 +7,29 @@ title: ECE 477 Team 5 Homepage
+
+A tuned grid of Hall effect sensors lies beneath the playing field of the Smart Air Hockey Table. These sensors measure disturbances in the magnetic field caused by a magnet embedded within the puck. Microcontrollers within the table read the values generated by these Hall effect sensors and process the data to get an accurate position of the puck in real-time.
+
+A grid of RGB LEDs also lies beneath the playing field of the Smart Air Hockey Table. These RGB LEDs operate as an optional backlight for the playing field with variable colors and intensity. However, each RGB LED can be individually controlled, allowing for almost limitless complex visuals. When combined with puck tracking and the game state, these visuals become reactive to the gameplay.
+
+Beside these innovative features, the Smart Air Hockey Table retains all the typical features consumers expect from a recreational air hockey table designed for residential use. This includes robust goal detection using photoresistors and an LCD integrated into the table for displaying the current score and game state.
+
+The dynamic nature of the reactive surface introduces novel game modes that transcend the boundaries of conventional air hockey. Despite its technologically advanced features, the smart air hockey table maintains a compact form factor, making it adaptable to a variety of flat surfaces, from regular tables to other suitable platforms. This convergence of timeless gameplay with modern technological advancements is poised to redefine the enjoyment of air hockey, offering enthusiasts an immersive and unparalleled gaming encounter.
 
 ## Project Specific Design Requirements (PSDRs)
 
 The design and development of the smart air hockey table will adhere to the
 following focused design requirements to ensure a successful and captivating
 gameplay experience:
 
-1. PSDR #1 (Hardware): An ability to control a grid of lights beneath the
-   playing surface to create complex visuals.
-2. PSDR #2 (Hardware & Software): An ability to accurately track the puck’s
-   position on the playing field in real time.
-3. PSDR #3 (Hardware): An ability to reliably detect goals scored by monitoring
-   the passage of the puck through the goal.
-4. PSDR #4 (Hardware): An ability to internally track the score of ongoing
-   games, which is then displayed with a display.
-5. PSDR #5 (Software): An ability to display dynamic light effects on the
-   playing surface based on game state by implementing software that synchronizes
-   isual effects with ongoing gameplay.
+1. PSDR #1 (Hardware): 
+An ability to control a grid of RGB LEDs beneath the playing surface with a refresh rate of [TODO].
+2. PSDR #2 (Hardware & Software):
+An ability to track the puck’s position on the playing field with a maximum average latency of [TODO].
```

Comment:
Implement a way of reading and processing an array of analog hall effect sensors using a "row-master" microcontroller

Create another PSDR that talks about sending this data to the main microcontroller.

TBD since we might change to IO expander, @benjamin-owen thoughts?

Reply:
To be honest, if we really are leaning toward nothing below the row-masters, we can just call them slave microcontrollers.